distribution/tools/plugin-cli/src/main/java/org/opensearch/plugins/InstallPluginCommand.java

```diff
@@ -748,13 +751,13 @@ private Path unzip(Path zip, Path pluginsDir) throws IOException, UserException
                 }
                 if (entry.isDirectory() == false) {
                     try (OutputStream out = Files.newOutputStream(targetFile)) {
+                        InputStream input = zipFile.getInputStream(entry);
```

Comment:
Does the input stream not need to be closed, including on error?

Reply:
Yes, I will add `input.close()`.

Reply:
Include it in the try-with-resources:

```
try (OutputStream out = Files.newOutputStream(targetFile);
    InputStream input = zipFile.getInputStream(entry)) {
...
}
```